Doxyfile

```diff
@@ -1,2976 +1,20 @@
-# Doxyfile 1.15.0
+PROJECT_NAME           = "ROOT"
+PROJECT_BRIEF          = "ROOT project"
+OUTPUT_DIRECTORY       = docs            # will be relative to CMake binary dir after configure
 
-# This file describes the settings to be used by the documentation system
-# Doxygen (www.doxygen.org) for a project.
-#
-# All text after a double hash (##) is considered a comment and is placed in
-# front of the TAG it is preceding.
-#
-# All text after a single hash (#) is considered a comment and will be ignored.
-# The format is:
-# TAG = value [value, ...]
-# For lists, items can also be appended using:
-# TAG += value [value, ...]
-# Values that contain spaces should be placed between quotes (\" \").
-#
-# Note:
-#
-# Use Doxygen to compare the used configuration file with the template
-# configuration file:
-# doxygen -x [configFile]
-# Use Doxygen to compare the used configuration file with the template
-# configuration file without replacing the environment variables or CMake type
-# replacement variables:
-# doxygen -x_noenv [configFile]
+INPUT                  = ./libROOT ./libROOT/libROOT ./ROOT/ROOT ./README.md .clang-format .clang-tidy
```

Comment:
Question: do we need to specify sub-directories if we are setting `RECURSIVE = YES` below?